srcpkgs/sway/template

```diff
@@ -1,25 +1,22 @@
 # Template file for 'sway'
 pkgname=sway
-version=0.15.2
+version=1.0
 revision=1
-build_style=cmake
-configure_args="-DCMAKE_INSTALL_SYSCONFDIR=/etc"
-hostmakedepends="asciidoc pkg-config wayland-devel"
-makedepends="gdk-pixbuf-devel json-c-devel libcap-devel pam-devel pango-devel
- wlc-devel"
-conf_files="/etc/sway/config /etc/pam.d/swaylock"
-short_desc="An i3-compatible window manager for Wayland"
+build_style=meson
+conf_files="/etc/sway/config"
+hostmakedepends="pkg-config wayland-devel scdoc git"
+makedepends="wlroots-devel wayland-devel wayland-protocols pcre-devel
+ json-c-devel pango-devel cairo-devel gdk-pixbuf-devel"
+depends="xorg-server-xwayland"
+short_desc="Tiling Wayland compositor compatible with i3"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="MIT"
-homepage="http://swaywm.org"
-distfiles="https://github.com/SirCmpwn/sway/archive/${version}.tar.gz"
-checksum=05526e3038d2a5490a64bd816f1f04d2a6c214ddc6182835312b273b40b737ae
+homepage="https://swaywm.org"
+distfiles="https://github.com/swaywm/${pkgname}/archive/${version}.tar.gz"
+checksum=24dafd0f1e630e97a5dd47233841adf856b665e2321d6207acfe6b3002d1bc56
 
-pre_configure() {
-	sed -i CMakeLists.txt -e "s;-Werror;-Wno-error;"
-}
 post_install() {
 	vlicense LICENSE
 	# must be setuid without logind!
-	chmod u+s ${DESTDIR}/usr/bin/sway
+	#chmod u+s ${DESTDIR}/usr/bin/sway
```

Comment:
Should this be uncommented? It is not that every system should have elogind installed in order to use Sway...

Reply:
I put this there as a place holder. Should be changed to an INSTALL script with setcap, if at all possible.

https://github.com/swaywm/sway/wiki/Running-Sway-without-systemd#direct-via-capabilities

Reply:
I place the setcap command in an INSTALL script, but I don't know how to check that the script is running correctly.